.buildkite/dagster-buildkite/dagster_buildkite/package_spec.py

```diff
@@ -253,7 +253,7 @@ def skip_reason(self) -> Optional[str]:
         """
         # If self._should_skip is not None, then the result is cached on self._skip_reason and we can return it.
         if self._should_skip is not None:
-            if self._should_skip is False:
+            if self._should_skip is True:
```

Comment:
lol but also can we just do if self._should_skip?

Reply:
I guess so, but I want to be as explicit as possible